<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addFlight_Dialog</class>
 <widget class="QDialog" name="addFlight_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>301</width>
     <height>363</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>航空公司(Flight)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="FFROM_lineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="FSTATE_comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="FTO_lineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>起飞时间(FTIME)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>飞机型号(FMODEL)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>起飞日期(FDATE)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>到达时间(ARRTIME)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="FMODEL_comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="displayFormat">
       <string>yyyy/mm/dd</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Flight_lineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>到达日期(ARRDATE)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>航班号(FID)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>飞往(FTO)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>飞机状态(FSTATE)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>起飞地(FFROM)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="FID_lineEdit"/>
    </item>
    <item row="9" column="1">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="displayFormat">
       <string>yyyy/mm/dd</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QTimeEdit" name="timeEdit_2">
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="post_Button">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FID_lineEdit</tabstop>
  <tabstop>Flight_lineEdit</tabstop>
  <tabstop>FMODEL_comboBox</tabstop>
  <tabstop>FSTATE_comboBox</tabstop>
  <tabstop>FFROM_lineEdit</tabstop>
  <tabstop>FTO_lineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>dateEdit_2</tabstop>
  <tabstop>timeEdit_2</tabstop>
  <tabstop>post_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
